src/api/config/initializers/flipper.rb

```diff
@@ -4,7 +4,8 @@
 ENABLED_FEATURE_TOGGLES = [
   { name: :request_show_redesign, description: 'Redesign of the request pages to improve the collaboration workflow' },
   { name: :content_moderation, description: 'Reporting inappropriate content' },
-  { name: :color_themes, description: 'Color themes' }
+  { name: :color_themes, description: 'Color themes' },
+  { name: :foster_collaboration, description: 'Foster collaboration' }
```

Comment:
I'm not sure one huge feature flag for the whole opportunity makes sense. Remember this is about customers selectively turning new things on and off. What if I want to try labels but don't want to get "version tracking" in my way.

Also the description should explain to me what I turn on and off here...

Reply:
Should we group the smaller features into foster collaboration flag then? I do see the reason to allow users to manage `version tracking` or `labels`, but for features like the report bug button, the contribution guide or security policy we could just use this flag

Reply:
Let's define `foster_collaboration` and `labels` feature flags here for now

Reply:
> Let's define `foster_collaboration` and `labels` feature flags here for now

Done.